src/configs/base_report_templates.c4m

```diff
@@ -164,6 +164,8 @@ report and subtract from it.
   key._IMAGE_LAST_TAG_TIME.use               = true
   key._IMAGE_STORAGE_METADATA.use            = true
   key._STORE_URI.use                         = true
+  key._INFERRED_TECH_STACKS.use              = true
```

Comment:
well need to add these to `crashoverride` report template

https://github.com/crashappsec/chalk/blob/f1ff5ded510171c9d4e025d06157929aafce1a4d/src/configs/crashoverride.c4m

Reply:
complete